File: app/src/main/java/no/nav/tilbakemeldingsmottak/consumer/oppgave/domain/EndreOppgaveRequestTo.java
package no.nav.tilbakemeldingsmottak.consumer.oppgave.domain;

import lombok.Builder;
import lombok.Value;

@Value
@Builder
public class EndreOppgaveRequestTo {
    String id;
    String tildeltEnhetsnr;
    String tema;
    String versjon;
    String aktivDato;
    String prioritet;
    String oppgavetype;
    String journalpostId;
    String status;
}

File: app/src/main/kotlin/no/nav/tilbakemeldingsmottak/consumer/oppgave/domain/EndreOppgaveRequestTo.kt
package no.nav.tilbakemeldingsmottak.consumer.oppgave.domain

data class EndreOppgaveRequestTo(
    val versjon: String,

    val id: String? = null,
    val tildeltEnhetsnr: String? = null,
    val tema: String? = null,
    val aktivDato: String? = null,
    val prioritet: String? = null,
    val oppgavetype: String? = null,
    val journalpostId: String? = null,
    val status: String? = null
)
